### ISSUE TITLE
Doesn't compile with latest SDK

### ISSUE THREAD

**Issue by wayneprice** [jpnurmi/flutter_spinbox#28]
See below error...

```
../../../FlutterSDK/.pub-cache/hosted/pub.dartlang.org/flutter_spinbox-0.5.0/lib/src/base_spin_box.dart:68:52: Error: A value of type 'bool' can't be returned from a function with return type 'KeyEventResult'.
 - 'KeyEventResult' is from 'package:flutter/src/widgets/focus_manager.dart' ('../../../FlutterSDK/packages/flutter/lib/src/widgets/focus_manager.dart').
    _focusNode = FocusNode(onKey: (node, event) => _handleKey(event));
```

**Comment by wayneprice** [comment]
I was using the master branch of flutter, and the return type of onKey needs to be KeyEventResult. That change hasn't propagated into the stable branch yet.

**Comment by jpnurmi** [comment]
Fixed by #29 - thanks!
